posthog/clickhouse/test/__snapshots__/test_schema.ambr

```diff
@@ -1377,7 +1386,10 @@
   sum(console_log_count) as console_log_count,
   sum(console_warn_count) as console_warn_count,
   sum(console_error_count) as console_error_count,
-  sum(size) as size
+  sum(size) as size,
+  -- we can count the number of kafka messages instead of sending it explicitly
+  count(*) as message_count,
```

Comment:
this is clever... that's not always a good thing 🤣 

Is it better to explicitly add `message_count: 1` in the plugin server to aid the future traveller?